libr/asm/arch/cris/gnu/cris-dis.c

```diff
@@ -201,39 +201,39 @@ get_opcode_entry (unsigned int insn,
       memset (opc_table, 0, 65536 * sizeof (const struct cris_opcode *));
 
       dip_prefixes
-	= malloc (65536 * sizeof (const struct cris_opcode **));
+	= malloc (65536 * sizeof (dip_prefixes[0]));
       if (!dip_prefixes) {
 	      return NULL;
       }
 
       memset (dip_prefixes, 0, 65536 * sizeof (dip_prefixes[0]));
 
       bdapq_m1_prefixes
-	= malloc (65536 * sizeof (const struct cris_opcode **));
+	= malloc (65536 * sizeof (bdapq_m1_prefixes[0]));
       if (!bdapq_m1_prefixes) {
 	      return NULL;
       }
 
       memset (bdapq_m1_prefixes, 0, 65536 * sizeof (bdapq_m1_prefixes[0]));
 
       bdapq_m2_prefixes
-	= malloc (65536 * sizeof (const struct cris_opcode **));
+	= malloc (65536 * sizeof (bdapq_m2_prefixes[0]));
       if (!bdapq_m2_prefixes) {
 	      return NULL;
       }
 
       memset (bdapq_m2_prefixes, 0, 65536 * sizeof (bdapq_m2_prefixes[0]));
 
       bdapq_m4_prefixes
-	= malloc (65536 * sizeof (const struct cris_opcode **));
+	= malloc (65536 * sizeof (bdapq_m4_prefixes[0]));
       if (!bdapq_m4_prefixes) {
 	      return NULL;
       }
 
       memset (bdapq_m4_prefixes, 0, 65536 * sizeof (bdapq_m4_prefixes[0]));
```

Comment:
use calloc nstead of malloc+memset

Reply:
anyway this is gnu code, dont touch it too much because it will break when we sync again